config/server.go

```diff
@@ -40,8 +46,41 @@ func (g *Global) SetDefaultGlobalConfigValues() error {
 	return nil
 }
 
-func generateRouterID() (uint32, error) {
-	addr, err := getLoopbackIP()
+func (g *Global) ReadGlobalConfig() error {
+
+	return _readGlobalConfig(g)
```

Comment:
IMHO just put the content of `_readGlobalConfig` here. Moreover from my point of view `_readGlobalConfig` is a bad name for a Go function since it starts with `_`. That read like python.

https://golang.org/doc/effective_go.html#mixed-caps